src/main/java/cm/ptks/craftflowers/storage/LanguageFile.java

```diff
@@ -0,0 +1,472 @@
+        FLOWER_CARROT_AGE_7 = language.getString("flower.CARROT_AGE.CARROT_7");
+
+        FLOWER_BARRIER = language.getString("flower.BARRIER");
+
+        // item
+        ITEM_SECOND_PREFIX = language.getString("item.second_prefix");
+
+        // gui
+        GUI_TITLE = language.getString("gui.title");
+        GUI_CREATE_FLOWER = language.getString("gui.create_flower");
+        GUI_CANT_CREATE_EMPTY_FLOWER = language.getString("gui.cant_create_empty_flower");
+        GUI_REMOVE_FROM_TOP = language.getString("gui.remove_from_top");
+        GUI_CLEAR_ALL = language.getString("gui.clear_all");
+        GUI_FLOWER_GROUP_TITLE_COLOR = language.getString("gui.flower_group_title_color");
+        GUI_GLASS_NAME_COLOR = language.getString("gui.glass_name_color");
+        GUI_PREVIOUS = language.getString("gui.previous");
+        GUI_NEXT = language.getString("gui.next");
+        GUI_SAVED_FLOWERPOTS = language.getString("gui.saved_flowerpots");
+        GUI_SAVED_FLOWERPOTS_NAME_COLOR = language.getString("gui.saved_flowerpots_name_color");
+
+        // action
+        ACTION_NO_PERMISSION_PLACE = language.getString("action.no_permission_place");
+        ACTION_MISSING_FOLLOWING_ITEMS = language.getString("action.missing_following_items");
+        ACTION_MISSING_FOLLOWING_ITEMS_LIST = language.getString("action.missing_following_items_list");
+        ACTION_NO_PERMISSION_EDIT = language.getString("action.no_permission_edit");
+        ACTION_OUTDATE_VERSION = language.getString("action.outdate_version");
+        ACTION_CURRENT_VERSION = language.getString("action.current_version");
+        ACTION_NEWEST_VERSION = language.getString("action.newest_version");
+        ACTION_DOWNLOAD_NEW_VERSION = language.getString("action.download_new_version");
+        ACTION_FAILED_CHECK_VERSION = language.getString("action.failed_check_version");
+
+    }
+
+    public static String getCommandVersion(String version, String color) {
+        return COMMAND_VERSION.replaceAll("%1", color + version);
+    }
+
+    public static String getCommandAuthor(String authors) {
+        return COMMAND_AUTHOR.replaceAll("%1", authors);
+    }
+
+    public static String getCommandWebsite(String website) {
+        return COMMAND_WEBSITE.replaceAll("%1", website);
+    }
+
+    public static String getCommandSuccessLoad(String args) {
+        return COMMAND_SUCCESS_LOAD.replaceAll("%1", args);
+    }
+
+    public static String getCommandCantDelete(String args) {
+        return COMMAND_CANT_DELETE.replaceAll("%1", args);
+    }
+
+    public static String getCommandSuccessDelete(String args) {
+        return COMMAND_SUCCESS_DELETE.replaceAll("%1", args);
+    }
+
+    public static String getFlowerInfoAge(int age) {
+        return FLOWER_INFO_AGE.replaceAll("%1", String.valueOf(age));
+    }
+
+    public static String getActionMissingFollowingItemsList(String arror, String flower, Integer integer) {
```

Comment:
There are two things wrong here.
First: arrow instead of arror

Second: Please don't overwrite the original value.
Use for example: string.replaceAll("example", "examplereplacement").replaceAll("example2", "examplereplacement2")